stage6/01-install-packages/00-packages

```diff
@@ -72,3 +72,6 @@ libzmq3-dev
 gnuplot
 xterm
 libusb-1.0
+build-essential
+libssl-dev
+libopencv-dev
```

Comment:
i think only libopencv-dev is missing. maybe check for the other two packages to not be included in other stages

Reply:
I started a test build w/o the 2 packages you mentioned. Will come back with an answear

Reply:
It looks like the other two packages where already installed! Removed them!